docs/local-development.md

```diff
@@ -63,6 +63,12 @@ $ vendor/bin/bref local --handler=my-function.php --file=event.json
 Hello Jane
 ```
 
+Or if your `serverless.yml` is not in root folder, use the `--config` parameter instead:
+```bash
+$ vendor/bin/bref local hello --config=path/to/your/serverless.yml --file=event.json
+Hello world
+```
```

Comment:
I would suggest we keep the documentation simple and not mention the option: instead we suggest it in the error message. That way users will try, fail and be able to fix the command themselves.

Reply:
ok, no problem, I remove this from the doc